vibes/vibed-ui/src/core/Timer.cpp

```
@@ -0,0 +1,39 @@
+
+Timer::Timer() {}
+
+Timer::~Timer() {}
+
+void Timer::start() {
+    started = true;
+    stopped = false;
+    startTime = std::chrono::steady_clock::now();
+}
+
+void Timer::stop() {
```

Cppcheck (2.20.0)

[style] The function 'stop' is never used.

(unusedFunction)